cmd/zed/zed.d/zed-functions.sh

```diff
@@ -29,8 +30,8 @@ ZED_FLOCK_FD=8
 #
 zed_check_cmd()
 {
-    local cmd
-    local rv=0
+    cmd=$*
+    rv=0
```

Comment:
These errors came from adding `#!/bin/sh` as the first line of the file. `warning: In POSIX sh, 'local' is undefined. [SC2039]` is the error that results for the local keyword. I could turn the warning off for the file, but `SC2039` is the generic undefined warning, so we'd be losing that coverage.